src/nm_add_vm.c

```diff
@@ -390,33 +385,29 @@ void nm_add_vm_to_db(nm_vm_t *vm, uint64_t mac,
         vm->usb_enable ? NM_ENABLE : NM_DISABLE, /* USB enabled */
         NM_DEFAULT_USBVER, /* set USB 3.0 by default */
         NM_DISABLE, /* disable 9pfs by default */
+        NM_DISABLE,
```

Comment:
Please add a comment what this disables (like others).

Reply:
Added